### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by bsquidwrd** [fbis251/overwatch_news_feed#1]
First off, this is awesome.

Second, how are you parsing the website and pushing it to Github? How often does it check for new updates to the webpage? Every hour?

**Comment by fbis251** [comment]
Thanks, I'm glad you like the project

Unfortunately the process isn't automated. I have a local copy of the generated feed added to my RSS reader. When I notice an update in the feed I copy the updated Atom file to the github project and make a new commit by hand although I'm looking for a way to easily automate creating new commits and pushing them.

I uploaded the scripts and the Go program I'm using to generate the feed here: https://github.com/fbis251/overwatch_feed_generator

It uses a combination of python + beautiful soup 4 to parse the HTML and go with gorilla/feeds to generate the final Atom feed.

I included the systemd service I'm running on my raspberry pi that checks for updates from the Overwatch patch notes page once per hour.

I hope this helps
